extensions/kafka-client/runtime/pom.xml

```diff
@@ -88,6 +88,18 @@
                     </capabilities>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>io.smallrye</groupId>
+                <artifactId>jandex-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>make-index</id>
+                        <goals>
+                            <goal>jandex</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
```

Comment:
We usually don't do this, but instead add the empty `beans.xml` file.

Reply:
For runtime modules of extensions, I've mostly seen the plugin usage.

Reply:
It works, but it's not the preferred way, so let's use the `beans.xml` way for new stuff please.

Thanks

Reply:
Finally, I went with adding individual classes to the index.

Reply:
Even better 😉